.github/ISSUE_TEMPLATE/project-proposal-template.md

```diff
@@ -20,6 +20,7 @@ Project Repository {REPO}
 - [ ] Technical Charter / Participation Agreement / Trademark docs approved & signed by project
 - [ ] Board Vote
 - [ ] Project made active in PCC by notifying formation team
+  - [ ] Create support ticket to add the project category (Incubating / Graduated) and software license to project the definition
```

Comment:
Small nit, I would not indent this, it is not a subtask just something that has to be done.